app/views/main/dashboard.html.erb

```diff
@@ -15,9 +15,38 @@
           <td>
             <div class="btn-group" role="group" aria-label="Validate or Remove Teacher">
               <%= button_to("Validate", validate_path(:id => teacher.id),
-                 class: 'btn btn-success', type: 'button') %>
-              <%= button_to("Deny", deny_path(:id => teacher.id),
-                 class: 'btn btn-danger', type: 'button') %>
+                class: 'btn btn-success', type: 'button') %>
+              <span>
+                <button class="btn btn-danger" type="button" data-toggle="modal" data-target="<%= "#denialModal" + teacher.id.to_s %>" >
```

Comment:
This is a personal preference, but it would be nice if we could use `.js-XXXX` class names instead of IDs. So this could simply be `.js-deniaModal`.